src/blocks/block-coauthor-avatar/block.json

```diff
@@ -0,0 +1,55 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "co-authors-plus/avatar",
+	"version": "1.0.0",
+	"title": "Co-Author Avatar",
+	"category": "theme",
+	"description": "Displays a small scale version of a co-author's avatar. Utilizes fallbacks from Gravatar so everyone has an avatar.",
+	"supports": {
+		"align": [ "left", "right", "center" ],
+		"html": false,
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"width": true,
+			"__experimentalSelector": "img, .block-editor-media-placeholder",
+			"__experimentalSkipSerialization": true,
+			"__experimentalDefaultControls": {
+				"color": false,
+				"radius": false,
+				"width": false
+			}
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true,
+			"__experimentalDefaultControls": {
+				"margin": false,
+				"padding": false
+			}
+		}
+	},
+	"usesContext": [
+		"co-authors-plus/author"
```

Comment:
@douglas-johnson This is a great feature I didn't know about before, and I like how easy it makes extending coauthors blocks as you documented in the README. Super cool!